src/openshift/component.ts

```diff
@@ -203,6 +205,63 @@ export class Component extends OpenShiftItem {
         return Component.dev(component, 'podman');
     }
 
+    @vsCommand('openshift.component.binding.add')
+    static async addBinding(component: ComponentWorkspaceFolder) {
```

Comment:
Show progress bar with notification until webview opened, currently there were no clue when clicked `Bind Service`

Reply:
I misunderstood at first, but it should be fixed now.